.github/workflows/ci.yaml

```diff
@@ -47,19 +43,19 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install Python dependencies
         run: |
-          python -m pip install setuptools_rust flake8 yapf flake8-quotes
+          python -m pip install maturin flake8 yapf==0.31.0 flake8-quotes
```

Comment:
Could we move this to a `python/requirements.txt` file?

Reply:
Packages like yapf and flake8 are used in CI to validate formatting, and aren't necessary to run ERDOS. Also, I don't think we need a `requirements.txt`, since the `pyproject.toml` specifies the dependencies.